.github/workflows/verifyimage.yml

```diff
@@ -99,16 +99,54 @@ jobs:
 
       - name: Verify ${{ matrix.target }}
         run: |
+          HOST_AND_PORT="${HOST_AND_PORT}"
           [ $(docker inspect ${{ matrix.target }}-test --format='{{.State.Running}}') = 'true' ]
           if grep -q  "nginx "<<< "${{ matrix.target }}"; then
-            curl -q -D headers.txt http://localhost:8080/?test=../../etc/passwd
+            echo "### nginx tests ###"
+
+            echo "Storing headers for sample attack..."
+            curl -q -D headers.txt "${HOST_AND_PORT}/?test=../../etc/passwd"
+            echo "...done"
+
+            echo "Check status 403..."
             grep -q "HTTP/1.1 403 Forbidden" headers.txt
+            echo "...yes"
```

Comment:
Maybe you want to combine `echo -e` with echo.
Or use `printf` in the first line without `\n` and then just doing `echo "yes"`, so it will show better in one line.